src/utils/time.ts

```diff
@@ -135,13 +134,11 @@ export function oneHourAfterTimestamp(timestamp: number) {
  * @see https://date-fns.org/docs/format
  */
 export function formatUtcSeconds(utcSeconds?: number | null) {
-  return utcSeconds
-    ? format(utcSeconds * ONE_SECOND_IN_MILLISECONDS, "p")
-    : "(unknown)";
+  return utcSeconds ? format(fromUnixTime(utcSeconds), "p") : "(unknown)";
 }
 
 export function getHoursAgoInSeconds(hours: number) {
-  const nowInSec = Date.now() / ONE_SECOND_IN_MILLISECONDS;
+  const nowInSec = getUnixTime(Date.now());
   return nowInSec - hours * ONE_HOUR_IN_SECONDS;
 }
```

Comment:
[NB] This could also be simplified using:

- https://date-fns.org/v2.21.3/docs/subHours

```typescript
export const getHoursAgoInSeconds = (hours: number) => getUnixTime(subHours(Date.now(), hours));
```

Reply:
realized that this function is not used anywhere but in `getDaysAgoInSeconds`, so instead of using `subHours`, used `subDays` there and removed `getHoursAgoInSeconds`: https://github.com/sparkletown/sparkle/pull/1426/commits/64e2b9d2c1f4b2cf715ff86330b85947d012a966